config/routes.rb

```diff
@@ -4,6 +4,8 @@
   resources :articles do 
     resources :comments
   end
+
+  jsonapi_resources :rental
```

Comment:
General convention is for plural resources in Rails routes, as well as the accompanying controller. I'd suggest changing to `:rentals` here and `RentalsController`.